src/ceph-daemon

```diff
@@ -1131,20 +1129,36 @@ def command_shell():
         mounts[pathify(args.config)] = '/etc/ceph/ceph.conf:z'
     if args.keyring:
         mounts[pathify(args.keyring)] = '/etc/ceph/ceph.keyring:z'
+    podman_args = ['--privileged']
+    if args.command:
+        command = args.command
+    else:
+        command = ['bash']
+        podman_args += '-it'
```

Comment:
just out of curiosity, why don't you also pass `'--env', 'LANG=C'` in this case?